packages/plugins/graphql/server/src/services/utils/playground.ts

```diff
@@ -1,12 +1,17 @@
 import { Context } from '../types';
 
+// TODO: deprecate and remove this because it is only used to determine if we need helmet security exceptions
+// stores the state of the Apollo landingPage (playground)
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 export default (ctx: Context) => {
+  let enabled = false;
+
   return {
+    setEnabled(val: boolean) {
```

Comment:
Should we mark those functions as deprecated/experimental if they're meant to be removed later on?

Reply:
(I'm referring to the TODO comment above)

Reply:
We don't have a replacement for them at the moment, it's probably going to take a refactoring to be able to inject exceptions into the security middleware.  So for now I'm just going to leave it alone even though it's a bit ugly, it would be a lot of change for this one little problem.